{{cookiecutter.project_dirname}}/.gitignore

```diff
@@ -168,15 +168,22 @@ cython_debug/
 /static/
 media/
 
+# Ruff
+.ruff_cache
+
 # Others
-*.bak
 .bash_history
-.local
 .ipython
+.kolo
+.local
+.tfscan
+.trivy
```

Comment:
```suggestion
```
Move this two rows in https://github.com/20tab/django-continuous-delivery/pull/250

Reply:
okk